src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -5692,7 +5703,212 @@ private StatementAst DataStatementRule(Token dataToken)
 
         #region Pipelines
 
-        private PipelineBaseAst PipelineRule()
+        private PipelineBaseAst PipelineChainRule()
+        {
+            // G  pipeline-chain:
+            // G      pipeline
+            // G      pipeline-chain chain-operator pipeline
```

Comment:
By looking at the compiler changes, this should be `pipeline chain-operator pipeline-chain`.

Reply:
I think with left-associativity, which is now also properly honoured by the compiler, this should be accurate, yes?